### PR TITLE
[Doc]Add tutorial document for qwen-VL-Dense

### PR DESCRIPTION
**Pull request by MrZ20**
### What this PR does / why we need it?
This document employs the qwen3-vl-8b model and qwen2.5-vl-32b to demonstrate the primary verification steps for the Qwen-VL series dense models, including supported features, feature configuration, environment preparation, NPU deployment, and accuracy and performance evaluation.

### How was this patch tested?

- vLLM version: v0.12.0
- vLLM main: https://github.com/vllm-project/vllm/commit/ad32e3e19ccf0526cb6744a5fed09a138a5fb2f9

### CODE REVIEW

**Comment by MrZ20** [comment]
@MengqingCao

**Comment by MrZ20** [comment]
@MengqingCao Review required

File: docs/source/tutorials/single_npu_qwen2.5_vl_7B.md [hunk 1]
```diff
@@ -0,0 +1,194 @@
+# Single NPU (Qwen2.5-VL 7B)
```
Comment (MengqingCao) [inline docs/source/tutorials/single_npu_qwen2.5_vl_7B.md]: There is already a doc for Qwen2.5-VL 7B, why adding this again here? and if you want to change some content of it, you should modify the original one, instead of create a now doc

File: docs/source/tutorials/single_npu_qwen3_vl.md [hunk 2]
```diff
@@ -45,14 +47,15 @@ Run the following script to execute offline inference on a single NPU:
 
 ```bash
 pip install qwen_vl_utils --extra-index-url https://download.pytorch.org/whl/cpu/
+pip install transformers>=4.57.0
```
Comment (MengqingCao) [inline docs/source/tutorials/single_npu_qwen3_vl.md]: This is done in vllm settingup period. Thus I think we could just remove it

File: docs/source/tutorials/index.md [hunk 3]
```diff
@@ -5,6 +5,7 @@
 :maxdepth: 1
 single_npu
 single_npu_qwen2.5_vl
+single_npu_qwen3_vl.md
```
Comment (1092626063) [inline docs/source/tutorials/index.md]: The file name can be `docs/source/tutorials/Qwen3-VL.md`, just like `DeepSeek-V3.2-Exp`

File: docs/source/tutorials/Qwen-VL-Dense.md [hunk 4]
```diff
@@ -0,0 +1,258 @@
...
+
+This document will show the main verification steps of the model, including supported features, feature configuration, environment preparation, single-NPU deployment, accuracy and performance evaluation.
+
+The `Qwen3-VL 8B` model is first supported in `vllm-ascend:v0.11.0rc0`.
+
+**Note on Model Compatibility**: All steps and configurations provided in this guide, including **Offline Inference** and **Online Serving**, are also **applicable** to the **Qwen2.5-VL 7B** model. To use this model, simply replace the model path `Qwen/Qwen3-VL-8B-Instruct` in the configuration or code with `Qwen/Qwen2.5-VL-7B-Instruct`.
+
+## Supported Features
+
+Refer to [supported features](../user_guide/support_matrix/supported_models.md) to get the model's supported feature matrix.
```
Comment (1092626063) [inline docs/source/tutorials/Qwen-VL-Dense.md:13]: do not forget to update supported_models.md

File: docs/source/tutorials/Qwen-VL-Dense.md [hunk 5]
```diff
@@ -0,0 +1,258 @@
...
+
+Refer to [feature guide](../user_guide/feature_guide/index.md) to get the feature's configuration.
+
+## Environment Preparation
```
Comment (1092626063) [inline docs/source/tutorials/Qwen-VL-Dense.md:17]: we should provide model weights in `Environment Preparation`

File: docs/source/tutorials/Qwen-VL-Dense.md [hunk 6]
```diff
@@ -0,0 +1,258 @@
...
+```bash
+pip install qwen_vl_utils --extra-index-url https://download.pytorch.org/whl/cpu/
+```
+
+```python
+from transformers import AutoProcessor
+from vllm import LLM, SamplingParams
+from qwen_vl_utils import process_vision_info
+
+MODEL_PATH = "Qwen/Qwen3-VL-8B-Instruct"
```
Comment (1092626063) [inline docs/source/tutorials/Qwen-VL-Dense.md:135]: Do we have quantized version？It's better to use quantized weights as development example

File: docs/source/tutorials/Qwen-VL-Dense.md (hunk 4 above)
Comment (MrZ20) [reply docs/source/tutorials/Qwen-VL-Dense.md:13]: Added.

File: docs/source/tutorials/Qwen-VL-Dense.md (hunk 5 above)
Comment (MrZ20) [reply docs/source/tutorials/Qwen-VL-Dense.md:17]: Added.

File: docs/source/tutorials/Qwen-VL-Dense.md (hunk 6 above)
Comment (MrZ20) [reply docs/source/tutorials/Qwen-VL-Dense.md:135]: There is currently no quantified version of this model.

File: docs/source/tutorials/Qwen3-VL-8B.md [hunk 7]
```diff
@@ -1,15 +1,36 @@
-# Single NPU (Qwen2.5-VL-7B)
+# Single NPU (Qwen3-VL-8B)
```
Comment (wangxiyuan) [inline docs/source/tutorials/Qwen3-VL-8B.md]: name to `Qwen3-VL-8B`, remove `Single NPU`

**Comment by MrZ20** [comment]
Document request merge. @wangxiyuan

File: docs/source/tutorials/Qwen-VL-Dense.md [hunk 8]
```diff
@@ -194,3 +197,51 @@ INFO 03-12 11:16:50 logger.py:39] Received request chatcmpl-92148a41eca64b6d82d3
...
+- `serve`: Benchmark the online serving throughput.
+- `throughput`: Benchmark offline inference throughput.
+
+Take the `serve` as an example. Run the code as follows.
+
+```shell
+vllm bench serve --model Qwen/Qwen3-VL-8B-Instruct  --dataset-name random --random-input 200 --num-prompt 200 --request-rate 1 --save-result --result-dir ./
+```
+
+After about several minutes, you can get the performance evaluation result.
```
Comment (gcanlin) [inline docs/source/tutorials/Qwen-VL-Dense.md:577]: Could you please add the result of performance here like accuracy evaluation? BTW, I think that we can add the link to our related CI about Qwen3-VL, which shows that we continue monitor the model and always ensure its perf and correctness.

File: docs/source/tutorials/index.md [hunk 9]
```diff
@@ -4,7 +4,7 @@
 :caption: Deployment
 :maxdepth: 1
 single_npu
-single_npu_qwen2.5_vl
+Qwen3-VL-8B.md
```
Comment (gcanlin) [inline docs/source/tutorials/index.md]: I think that we can gather all guides about Qwen3-VL in this markdown file including single and multi-card. If so, we can remove `-8B`. But for now, let's add the smaller models first, like `2B`, `4B` and `8B`. Finally, all models in [Qwen3-VL Collection](https://huggingface.co/collections/Qwen/qwen3-vl) can be deployed by referring to this single document. What do you think? cc @Yikun , @wangxiyuan

File: docs/source/tutorials/Qwen-VL-Dense.md (hunk 8 above)
Comment (gcanlin) [reply docs/source/tutorials/Qwen-VL-Dense.md:577]: Would be better if you could try the real datasets instead of random datasets. Like:

```
vllm bench serve --backend openai-chat --model Qwen/Qwen3-VL-8B-Instruct --endpoint /v1/chat/completions --dataset-name hf --dataset-path lmarena-ai/VisionArena-Chat --hf-split train --num-prompts 1000
```

File: docs/source/tutorials/Qwen-VL-Dense.md [hunk 10]
```diff
@@ -194,3 +197,51 @@ INFO 03-12 11:16:50 logger.py:39] Received request chatcmpl-92148a41eca64b6d82d3
...
+
+```shell
+lm_eval \
+    --model vllm-vlm \
+    --model_args pretrained=/root/.cache/modelscope/hub/models/Qwen/Qwen3-VL-8B-Instruct,max_model_len=8192,gpu_memory_utilization=0.7 \
+    --tasks mmmu_val \
+    --batch_size 32 \
+    --apply_chat_template \
+    --trust_remote_code \
+    --output_path ./results
```
Comment (gcanlin) [inline docs/source/tutorials/Qwen-VL-Dense.md:501]: Just a reminder. We should make sure the parameters consistent with our CI.

File: docs/source/tutorials/Qwen-VL-Dense.md (hunk 8 above)
Comment (1092626063) [reply docs/source/tutorials/Qwen-VL-Dense.md:577]: Thank you for your advise, we have to check：
1. Whether performance data can be presented needs to be discussed.
2. What models will our CI execute?
3. Use real datasets is a good idea, but whether to use real datasets, i think it depends on question 1.
cc @wangxiyuan

File: docs/source/tutorials/index.md (hunk 9 above)
Comment (Yikun) [reply docs/source/tutorials/index.md]: +1 Gather all in one doc, it's OK.

File: docs/source/tutorials/Qwen-VL-Dense.md [hunk 11]
```diff
@@ -41,23 +73,26 @@ export PYTORCH_NPU_ALLOC_CONF=max_split_size_mb:256
...
+
 Run the following script to execute offline inference on a single NPU:
 
 ```bash
 pip install qwen_vl_utils --extra-index-url https://download.pytorch.org/whl/cpu/
 ```
 
-Create a python script with the following content:
-
 ```python
```
Comment (Yikun) [inline docs/source/tutorials/Qwen-VL-Dense.md:130]: Please add a multi card example tab here `Qwen3-VL-32B-Instruct`

File: docs/source/tutorials/Qwen-VL-Dense.md [hunk 12]
```diff
@@ -1,15 +1,47 @@
...
+
+This document will show the main verification steps of the model, including supported features, feature configuration, environment preparation, NPU deployment, accuracy and performance evaluation.
+
+The steps and configurations in this document apply to the following models:
+- Single NPU Support:
+  - `Qwen2.5-VL-7B-Instruct`
+  - `Qwen3-VL-2B-Instruct`
+  - `Qwen3-VL-4B-Instruct`
+  - `Qwen3-VL-8B-Instruct`
+- Multi-NPU Support (Requires 4 NPUs):
```
Comment (Yikun) [inline docs/source/tutorials/Qwen-VL-Dense.md]: Why 4 NPUs? I guess 2 NPU is enough, is there any bug here?

**Review by Yikun — APPROVED** [review]
Otherwise LGTM
